tests/example_stac/example-item_sci-extension.json

```diff
@@ -0,0 +1,70 @@
+{
+    "stac_version": "1.0.0",
+    "stac_extensions": [
+      "scientific",
```

Comment:
The extension ids need to be fully qualified in 1.0.0 I think?

Reply:
You're absolutely right

Reply:
I'm going to fix this in the tests but I wonder if more validation isn't desirable in the model itself to avoid this kind of oversight

Reply:
It looks like JSON Schema now requires it to be an `iri` https://github.com/radiantearth/stac-spec/blob/789e90fba4c7119bc925d26f75f47bacf44c91e6/item-spec/json-schema/item.json#L105

So maybe use `AnyUrl`? https://github.com/radiantearth/stac-spec/blob/789e90fba4c7119bc925d26f75f47bacf44c91e6/item-spec/json-schema/item.json#L105